juniper_official/Solutions/Icmp-outlier/icmp.rule

```diff
@@ -0,0 +1,252 @@
+/*
+ * Sends ICMP probes to user specified destination host and notifies anomalies 
+ * when round trip time average response is above static or dynamic thresholds.
+ * And also detects outlier across the devices in device group ML algorithms.
+ * Three inputs control detection
+ * 
+ *   1) host_var, is a destination IP or host name where ICMP probes send 
```

Comment:
Please use "-" instead of "_" for variable definitions.

Reply:
Modified